proxy/config.php

```diff
@@ -0,0 +1,182 @@
+<?php
```

Comment:
@Edzelopez is there a reason to commit this when standard practice upstream is that this file is treated as a secret and therefor not committed?